src/Check/SecurityAdvisory.php

```diff
@@ -51,19 +49,14 @@ public function __construct($lockFilePath = null)
             ));
         }
 
-        if (! $lockFilePath) {
+        if ($lockFilePath === null) {
```

Comment:
```suggestion
        if ($lockFilePath === null || $lockFilePath === '') {
```

Maybe also check for an empty string here